app/controllers/api/admin/users_controller.rb

```diff
@@ -19,6 +21,14 @@ def destroy
 
   private
 
+  def fetch_entries
+    if params[:locked] == 'true'
```

Comment:
how about introducing the param converter concern? https://github.com/hitobito/hitobito/blob/master/app/controllers/concerns/param_converters.rb#L12 :man_dancing: